docs/tutorials/error_analysis.rst

```diff
@@ -0,0 +1,1199 @@
+**************
+Error analysis
+**************
+
+This tutorial describes how to propagate (a) the statistical uncertainties on
+the parameters of a model that was fit to a time series of scintillation
+velocities to (b) uncertainties on the physical parameters that describe the
+pulsar system and the scattering screen. The model assumes a pulsar on a
+circular orbit whose radiation is scattered by a single one-dimensional screen.
+The tutorial builds upon a :doc:`preceding tutorial <infer_phys_pars>` that
+described in more detail how the physical parameters can be inferred from the
+model parameters. Like in the second part of that tutorial, this
+error-propagation tutorial assumes there are some constraints on the distance
+to the pulsar. The tutorial uses the fit results generated in the tutorial on
+:doc:`fitting scintillation velocities <fit_velocities>`. These fit results are
+available for download:
+:download:`fit-results-J0437.npz <../data/fit-results-J0437.npz>`
+
+We will do error propagation using two methods:
+
+- Monte Carlo sampling, using Astropy's :py:mod:`~astropy.uncertainty` module.
+- Linear error propagation, using the :py:mod:`uncertainties` package.
```

Comment:
This link does not currently work. It would need a direct link, or if you refer to its methods/classes, an entry to `conf.py` defining intersphinx links -- e.g., https://github.com/mhvk/baseband-tasks/blob/b8b139de652f6d84de227898695777dfcdd9f8a3/docs/conf.py#L67-L72

Reply:
Added manual links for all references to the `uncertainties` package and its components throughout the document. It requires a bit of a hack each time to get the  typesetting (monospace) consistent with the links automatically created by intersphinx.